CHANGELOG.md

```diff
@@ -1,3 +1,615 @@
+# 2.0.0-rc.76 (2022-08-30)
+
+
+### Bug Fixes
```

Comment:
please remove this file from PR.

Reply:
Updated PR with additional screenshots

Reply:
@bpateldx My bad. I meant to remove changes in this file from PR.
sorry for the inaccuracy

Reply:
@AlexVarchuk Just to be clear, you're simply asking to revert change log file back to your master version, correct? I assume you would be updating the changelog file upon release.

Reply:
@bpateldx yes, please revert `CHANGELOG.md` file to the master version. We'll update it when making the next release. Thx

Reply:
Done, please review.